build/local_install_or_update

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Generic setup
+set -e
+
+# Ensure we have a $FLEPI_PATH
+if [ -z "${FLEPI_PATH}" ]; then
+    export FLEPI_PATH=$(realpath "$(pwd)")
+    if [ "$(basename "$FLEPI_PATH")" != "flepiMoP" ]; then 
+    echo "Could not find flepiMoP repository in your current directory."
+    exit 1
+    fi
+    echo "Using '$FLEPI_PATH' for \$FLEPI_PATH."
+fi
+
+# Test that flepiMoP is located there, exit if not
+if [ ! -d "$FLEPI_PATH" ]; then
+    echo "Could not find flepiMoP repository in your current directory. Please make sure you have correctly cloned flepiMoP in this directory."
+    exit 1
+fi
+
+# Ensure that conda environment is named flepimop-env
+if [ -z "${FLEPI_CONDA}" ]; then
+    export FLEPI_CONDA="flepimop-env"
+    echo "Using '$FLEPI_CONDA' for \$FLEPI_CONDA."
+fi
+
+# Check that the name of the FLEPI_CONDA variable matches the environment that has been set up, exit if not
+FLEPI_CONDA_ENV_MATCHES=$( conda info --envs | awk '{print $1}' | grep -x "$FLEPI_CONDA" | wc -l )
+if [ "$FLEPI_CONDA_ENV_MATCHES" -eq 0 ]; then
+    echo "Issue with conda environment: ensure you have set up your conda environment and it has the correct name."
+    exit 1
+fi
+
+# Load the conda environment
+conda activate $FLEPI_CONDA
```

Comment:
```suggestion
# Load the conda environment
eval "$(conda shell.bash hook)"
conda activate $FLEPI_CONDA
```
I had to add this line to get this to run on my system but maybe it's because of my setup. @emprzy were you able to successfully run the script without this line?

Reply:
Ah that's a good catch. I think I didn't realize that conda needs to be initialized in every new shell session (?), so if you're installing flepiMoP in the same session that you originally installed/initialized conda, then the code will work. Otherwise, you'll need this initialization line.